package.yaml

```diff
@@ -11,7 +11,7 @@ extra-source-files:
 
 license: MIT
 
-tested-with: GHC ==9.2.4 || ==9.4.2
+tested-with: ==9.4.8 || ==9.6.3
```

Comment:
No more 9.2 support?

Reply:
i can add it back in. i just thought it of it as a sliding window, i guess.